code/modules/projectiles/ammunition.dm

```diff
@@ -27,7 +27,7 @@
 /obj/item/ammo_casing/update_icon()
 	..()
 	icon_state = "[initial(icon_state)][BB ? "-live" : ""]"
-	desc = "[initial(desc)][BB ? "" : " This one is spent"]"
+	desc += "[BB ? "" : " This one is spent."]"

```

Comment:
Revert this line.

Otherwise:
```
update_icon()
update_icon()
update_icon()
```
would make the desc:
" This one is spent. This one is spent. This on is spent"